### DISCUSSION TITLE
Experience with managed Postgres on Azure Flexible Postgres

### DISCUSSION THREAD

**Discussion by tom-bywild** [supabase/supabase#22287] [Questions]
I'm currently working on hosting Supabase on Azure, specifically hosting the various docker images via Azure Container Apps. So far the setup looks good, but I hit a blocker when trying to use Azure Flexible Postgres for the database.

To use Flexible Postgres, I collected the various init SQL scripts from supabase/postgres and supabase/supabase to prepare an empty db. The issue is that BYPASSRLS is used in those scripts to alter the supabase_admin role, but it can't be used on Flexible Postgres due to constraints by Microsoft.

I therefore wanted to ask if

- someone has managed to use Azure Flexible Postgres as database at all for a self-hosted Supabase
- if, by any chance, there is a solution to actually not use BYPASSRLS (I'd be surprised if that's the case)

Thanks!

**Comment by Sulomus** [comment]
@tom-bywild did you find any solution? :)
